test/e2e/satokensigner_test.go

```diff
@@ -1,46 +0,0 @@
-package e2e
-
-import (
-	"context"
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clientcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
-
-	configv1 "github.com/openshift/api/config/v1"
-	configclient "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
-	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
-	test "github.com/openshift/cluster-kube-controller-manager-operator/test/library"
-)
-
-func TestSATokenSignerControllerSyncCerts(t *testing.T) {
```

Comment:
I think it would be better if we could keep the test
- we can test the current change
- the syncing of `service-account-private-key` is not really simple, so testing the secret recreation still has benefits